apollo-portal/src/main/resources/static/scripts/controller/role/ClusterNamespaceRoleController.js

```diff
@@ -0,0 +1,166 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+role_module.controller('ClusterNamespaceRoleController',
+    ['$scope', '$location', '$window', '$translate', 'toastr', 'AppService', 'UserService', 'AppUtil', 'EnvService',
+      'PermissionService',
+      function ($scope, $location, $window, $translate, toastr, AppService, UserService, AppUtil, EnvService,
+          PermissionService) {
+
+        var params = AppUtil.parseParams($location.$$url);
+        $scope.pageContext = {
+          appId: params.appid,
+          env: params.env,
+          clusterName: params.clusterName
+        };
+
+        $scope.modifyRoleSubmitBtnDisabled = false;
+        $scope.ReleaseRoleSubmitBtnDisabled = false;
+
+        $scope.releaseRoleWidgetId = 'releaseRoleWidgetId';
+        $scope.modifyRoleWidgetId = 'modifyRoleWidgetId';
+
+        PermissionService.init_cluster_ns_permission($scope.pageContext.appId, $scope.pageContext.env, $scope.pageContext.clusterName)
+        .then(function (result) {
+
+        }, function (result) {
+          toastr.warning(AppUtil.errorMsg(result), $translate.instant('Cluster.Role.InitClusterPermissionError'));
+        });
+
+        PermissionService.has_assign_user_permission($scope.pageContext.appId)
+        .then(function (result) {
+          $scope.hasAssignUserPermission = result.hasPermission;
+        }, function (result) {
+
+        });
+
+        PermissionService.get_cluster_ns_role_users($scope.pageContext.appId,
+            $scope.pageContext.env, $scope.pageContext.clusterName)
+        .then(function (result) {
+          $scope.rolesAssignedUsers = result;
+        }, function (result) {
+          toastr.error(AppUtil.errorMsg(result), $translate.instant('Cluster.Role.GetGrantUserError'));
+        });
+
+        $scope.assignRoleToUser = function (roleType) {
+          if ("ReleaseNamespacesInCluster" === roleType) {
+            var user = $('.' + $scope.releaseRoleWidgetId).select2('data')[0];
+            if (!user) {
+              toastr.warning($translate.instant('Cluster.Role.PleaseChooseUser'));
+              return;
+            }
+            $scope.ReleaseRoleSubmitBtnDisabled = true;
+            var toAssignReleaseNamespacesInClusterRoleUser = user.id;
+
+            var assignReleaseNamespacesInClusterRoleFunc = function (appId, env, clusterName, user) {
+              return PermissionService.assign_release_cluster_ns_role(appId, env, clusterName, user);
+            };
+
+            assignReleaseNamespacesInClusterRoleFunc(
+                $scope.pageContext.appId,
+                $scope.pageContext.env,
+                $scope.pageContext.clusterName,
+                toAssignReleaseNamespacesInClusterRoleUser
+            ).then(function () {
+              toastr.success($translate.instant('Cluster.Role.Added'));
```

Comment:
Looks like some labels are not properly translated.

![image](https://github.com/user-attachments/assets/02b39f5f-2c46-4926-a2ac-2cfeb2fb508d)

![image](https://github.com/user-attachments/assets/ae3e4821-2db2-4185-9195-63f04bc9e7e2)